src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -1,10 +1,10 @@
 # This file defines the default (English) variants of all of the internationalised messages
 pscanalpha.desc = Passive Scan Rules - alpha
 
-pscanalpha.blanktarget.name=Link uses _blank target
-pscanalpha.blanktarget.desc=At least one link on this page uses a target of "_blank" without using either keyword "noopener" or "noreferrer" in the "rel" attribute, which allows the target page to take control of this page.
-pscanalpha.blanktarget.refs=https://dev.to/ben/the-targetblank-vulnerability-by-example\nhttps://mathiasbynens.github.io/rel-noopener/\nhttps://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c\n
-pscanalpha.blanktarget.soln=Do not use a target of "_blank", or if you have to then also add the attribute: rel="noopener".
+pscanalpha.linktarget.name=Reverse tabnabbing
+pscanalpha.linktarget.desc=At least one link on this page is vulnerable to Reverse tabnabbing as it uses a target attribute without using both of the "noopener" and "noreferrer" keywords in the "rel" attribute, which allows the target page to take control of this page.
+pscanalpha.linktarget.refs=https://www.owasp.org/index.php/Reverse_Tabnabbing\nhttps://dev.to/ben/the-targetblank-vulnerability-by-example\nhttps://mathiasbynens.github.io/rel-noopener/\nhttps://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c\n
+pscanalpha.linktarget.soln=Do not use a target attribute, or if you have to then also add the attribute: rel="noopener noreferrer".
```

Comment:
Kinda nit picky, but these should be moved alphabetically within the file as well ;)